Tetris/Tetris.h

```diff
@@ -50,6 +48,9 @@ class Tetris {
     bool isPaused;                                  // is a game paused?
```

Comment:
number of flags is increasing - it may make sense to model states in an enum class ...like "idle", "playing", "pausing", "quitted" ...

Reply:
introduced proper state model and implemented state machine